### DISCUSSION TITLE
Ideas de como organizareis vuestra parte de la presentación

### DISCUSSION THREAD

**Discussion by Ballwictb** [IsraC0d33/Proyecto1Marcas#56] [Ideas]
Decir como presentareis cada uno vuestra parte de tiempo para que los demás nos podamos adaptar y a la vez sepamos mas o menos como planteareis vuestra parte.

[Presentacion](https://aulavirtual33.educa.madrid.org/ies.laarboleda.alcorcon/mod/jitsi/view.php?id=34292)

@MuiGt @IsraC0d33 @MarcosAcedoLlerena @MarioGalanEscudero @roberto-ortizperez

**Comment by IsraC0d33** [comment]
Le explicare mi codigo de html con las clase usadas y la estructura, admeas de contarle mi meta, 
luego le explicare un bot de analisis de codigo, los secrets 
y explicar que he arreglado incidencias de codacy

**Comment by Ballwictb** [comment]
Explicar mi meta y la implementación de iconos,arreglo del sonarcloud,implemntacion de action con israel,implemntacion de bots con israel,cambios del codigo de html y css y js,creacion del releases,implemntacion de los badges dinamicos y estaticos

**Comment by MarcosAcedoLlerena** [comment]
Explicar lo de css y como hice lo de las cookies y ns que mas

**Comment by MuiGt** [comment]
Ya que no he he hecho mucho, podría contar lo que hice de css (lo de implementar las banderas en la barra de navegación) y empezar la presentación como introducción para que podáis seguir vosotros, además de ir explicando los procedimientos que hemos ido utilizando durante el trabajo, si os parece bien
